lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -1802,19 +1802,12 @@ String? textAlignToCssValue(ui.TextAlign? align, ui.TextDirection textDirection)
     case ui.TextAlign.justify:
       return 'justify';
     case ui.TextAlign.end:
-      switch (textDirection) {
-        case ui.TextDirection.ltr:
-          return 'end';
-        case ui.TextDirection.rtl:
-          return 'left';
-      }
-    default: // including ui.TextAlign.start
-      switch (textDirection) {
-        case ui.TextDirection.ltr:
-          return null; // it's the default
-        case ui.TextDirection.rtl:
-          return 'right';
-      }
+      return textDirection == ui.TextDirection.ltr ? 'end' : 'left';
```

Comment:
Lol I know this is annoying, but the flutter style guide recommends always using `switch` with enums: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or--or--with-enum-values

Reply:
Done. As in all things there should have been exception if enum is clearly bool like i.e. don't see a third direction middle coming up, although maybe someone will do top to bottom :). Not as annoying as no dartfmt.